packages/browser/src/parsers.ts

```diff
@@ -67,48 +65,61 @@ export function eventFromPlainObject(
 /**
  * @hidden
  */
-export function eventFromStacktrace(stacktrace: TraceKitStackTrace): Event {
-  const exception = exceptionFromStacktrace(stacktrace);
-
+export function eventFromError(ex: Error): Event {
   return {
     exception: {
-      values: [exception],
+      values: [exceptionFromError(ex)],
     },
   };
 }
 
-/**
- * @hidden
- */
-export function prepareFramesForEvent(stack: StackFrame[]): StackFrame[] {
-  if (!stack.length) {
-    return [];
+/** Parses stack frames from an error */
+export function parseStackFrames(ex: Error & { framesToPop?: number; stacktrace?: string }): StackFrame[] {
+  // Access and store the stacktrace property before doing ANYTHING
+  // else to it because Opera is not very good at providing it
+  // reliably in other circumstances.
+  const stacktrace = ex.stacktrace || ex.stack || '';
+
+  const popSize = getPopSize(ex);
+
+  try {
+    // The order of the parsers in important
```

Comment:
If we're going to allow users to add parsers themselves, some sort of priority will be required and it should be possible with minimal impact on bundle.

Since this PR doesn't expose configurability to users yet, I added this comment as a warning because the ordering might not be obvious at this point!